README.md

```diff
@@ -1,2 +1,34 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
```

Comment:
Maybe this doc should just be two sections
- Prerequisites (stopping the app, creating the target storage class, etc)
- Command line usage

More end-to-end documentation will be in rook (for flex migration) or in a different .md for in-tree

Reply:
Currently, it has 4 sections

1. # What is persistent-volume-migrator?
2. # Getting Started
3.  ## Prerequisite
4. ## Steps to Migrate

You are suggesting removing 1st two sections but this is the main doc that the user will see.

Reply:
Looks good as it is now, the rook links were just strange